presto-product-tests/conf/presto/etc/jvm.config

```diff
@@ -10,13 +10,10 @@
 -XX:+UseG1GC
 -XX:+UseGCOverheadLimit
 -XX:+ExplicitGCInvokesConcurrent
--XX:+CMSClassUnloadingEnabled
--XX:+AggressiveOpts
 -XX:+HeapDumpOnOutOfMemoryError
 -XX:+ExitOnOutOfMemoryError
 -XX:ReservedCodeCacheSize=150M
 -DHADOOP_USER_NAME=hive
 -Duser.timezone=Asia/Kathmandu
--Xdebug 
--Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005
+#-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005
```

Comment:
Just remove?

Reply:
I wanted to leave that in for ease of use when someone locally wants to debug it. I don't care much though.

Reply:
Nice. Then I'd recommend updating to modern form (`-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005`). But i prefer removal anyway :)